### ISSUE TITLE
Redirect on form completion broken

### ISSUE THREAD

**Issue by mackina** [mautic/mautic#2511]
Bug Report

Description:

A form that is set to redirect to another URL on completion isn't redirecting. The submit button changes to "Please Wait" but no redirect takes place. The data does go in to Mautic. 

Browser receives the response from /form/submit but it doesn't execute. If I take the following line from the response and execute it in my browser then the redirect works : parent.postMessage...

I noticed that the /form/submit is loaded from the IP of my server (54.194.167.152/form/submit) rather than my subdomain (info.formisimo.com) so I'm wondering if there are any cross-domain JS issues.

If I use Developer Tools in Chrome and change the action on the form tag in the DOM to http://info.formisimo.com (from 54.194.167.152) then the redirect works.

Mautic Version: 2.11
PHP Version: 7.0.8

Steps to reproduce:
1.  Visit http://info.formisimo.com/formisimo-demo
2.  Fill in form, click submit.
3. No redirect takes place.

**Comment by mackina** [comment]
An update:

Upon further digging I noticed that the field cached_html in maforms referenced the IP rather than the subdomain. I wiped this field for the form that was breaking, and it then re-cached with the correct value and the form now works.

When I installed Mautic I had the site URL set as an IP until we moved our subdomain over. It could be that when changing the site URL from IP to the subdomain the cached HTML wasn't cleared.
